src/api/api.ts

```diff
@@ -1,4 +1,4 @@
-export const profileDataFetch = async (userId) => {
+export const profileDataFetch = async (userId: number) => {
```

Comment:
함수의 경우에는 행위가 앞쪽에 표현되는게 일반적인데요~
이렇게 표현해주는게 더 괜찮을 것 같아요!
```suggestion
export const fetchProfileData = async (userId: number) => {
```